easybuild/easyconfigs/a/ASE/ASE-3.21.1-foss-2020a-Python-3.8.2.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'PythonBundle'
+
+name = 'ASE'
+version = '3.21.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language.
+
+From version 3.20.1 we also include the ase-ext package, it contains optional reimplementations
+in C of functions in ASE.  ASE uses it automatically when installed."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', '-Python-%(pyver)s'),
+    ('Flask', '1.1.2', '-Python-%(pyver)s', ('GCCcore', '9.3.0')),
```

For this one, I'm not sure the toolchain part is needed, given that the `ASE` PR is at the `foss` level.
In any case, we can use `versionsuffix` again.
```suggestion
    ('Flask', '1.1.2', versionsuffix, ('GCCcore', '9.3.0')),
```
or just 
```suggestion
    ('Flask', '1.1.2', versionsuffix),
```